jest.config.js

```diff
@@ -17,9 +16,8 @@ module.exports = Object.assign({}, config, {
     '<rootDir>/packages/api-contract/build',
     '<rootDir>/packages/rpc-core/build',
     '<rootDir>/packages/rpc-provider/build',
-    '<rootDir>/packages/type-extrinsics/build',
+    '<rootDir>/packages/type-metadata/build/',
```

Comment:
```suggestion
    '<rootDir>/packages/type-metadata/build',
```

i don't think it changes anything, but for consistency's sake